exporter/redis.go

```diff
@@ -108,21 +102,12 @@ var (
 		"aof_last_write_status":        "aof_last_write_status",
 
 		// # Stats
-		"total_connections_received": "connections_received_total",
-		"total_commands_processed":   "commands_processed_total",
-		"instantaneous_ops_per_sec":  "instantaneous_ops_per_sec",
-		"total_net_input_bytes":      "net_input_bytes_total",
-		"total_net_output_bytes":     "net_output_bytes_total",
-		"instantaneous_input_kbps":   "instantaneous_input_kbps",
-		"instantaneous_output_kbps":  "instantaneous_output_kbps",
-		"rejected_connections":       "rejected_connections_total",
-		"expired_keys":               "expired_keys_total",
-		"evicted_keys":               "evicted_keys_total",
-		"keyspace_hits":              "keyspace_hits_total",
-		"keyspace_misses":            "keyspace_misses_total",
-		"pubsub_channels":            "pubsub_channels",
-		"pubsub_patterns":            "pubsub_patterns",
-		"latest_fork_usec":           "latest_fork_usec",
+		"instantaneous_ops_per_sec": "instantaneous_ops_per_sec",
+		"instantaneous_input_kbps":  "instantaneous_input_kbps",
```

Comment:
bytes/s is preferred

Reply:
Removed them, they can be calculated from other metrics.